flask_pagedown/__init__.py

```diff
@@ -3,11 +3,17 @@
 
 
 class _pagedown(object):
-    def include_pagedown(self):
+    def include_pagedown(self, urls=None):
```

Comment:
Accepting the urls as a list of two elements is confusing, as it does not make it clear in which order the libraries need to be given. I prefer two explicit arguments, `converter_js` and `sanitizer_js`, which leave no ambiguity. Also, instead of making them URLs, just make them filenames, which are then converted to URLs via `url_for`. That is more secure. Thanks!